playbooks/roles/ceph/tasks/storage-network.yml

```diff
@@ -0,0 +1,107 @@
+---
```

Comment:
Would you please take time to explain what this playbook does.
We already maintain net configs into https://github.com/shiftstack/dev-install/blob/main/playbooks/templates/dev-install_net_config.yaml.j2 and now it seems like it's also managed here.
The VIPs are handled by the installer, why are we creating it here?
Also, it's not taking in account the vars from https://github.com/shiftstack/dev-install/blob/main/playbooks/vars/defaults.yaml for the control plane, so if someone changes the vars, they'll have to override what's in playbooks/roles/ceph/tasks/main.yaml... not ideal and user friendly.